btc_parser/sources/utils.move

```diff
@@ -0,0 +1,110 @@
+module btc_parser::utils;
+use std::u64::do;
+use std::hash::sha2_256;
+
+
+#[test_only]
+use std::unit_test::assert_eq;
+
+#[error]
+const EOutOfBounds: vector<u8> = b"Slice out of bounds";
+
+public fun le_vec_to_number(v: vector<u8>): u64 {
```

Comment:
add a comment explaining what this function does. Its a module, meant to be reused by others